examples/network/edgeStyles/selfReference.html

```diff
@@ -72,10 +148,16 @@
                         }
                     }
                 });
+
+                edges.updateOnly({
+                    id: 3,
+                    selfReference: {
+                        fullRender: false
+                    }
+                });
```

Comment:
You set the same thing over and over again. Configure it once when the edge is initially created.